DadJokes/JokeStore.swift

```diff
@@ -30,9 +30,14 @@ struct JokeStore {
   
   /// we NEED more jokes here!!!
   private var storage: [Joke] = [
-    Joke(description: "Why don't programmers like being outside?", punchline: "Because there's too many bugs"),
-    Joke(description: "Why don't bachelors like Git?", punchline: "Because they are afraid to commit."),
-    Joke(description: "What does the father play with his son?", punchline: "Try Catch"),
+    Joke(description: "You should be careful with functions that return a value...",
+         punchline: "In fact, I would just a void them."),
```

Comment:
Looks better on a small screen, no?

Reply:
Shouldn't it actually read, "don't return a value" since void functions do not return a value?  Godo job tho!